backend/fief/services/email/sendgrid.py

```diff
@@ -0,0 +1,34 @@
+from typing import Optional, Tuple
+
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+from fief.services.email.base import EmailProvider, SendEmailError, format_address
+
+
+class Sendgrid(EmailProvider):
+    def __init__(self, server_token: str) -> None:
+        self._client = SendGridAPIClient(api_key=server_token)
+
+    def send_email(
+        self,
+        *,
+        sender: Tuple[str, Optional[str]],
+        recipient: Tuple[str, Optional[str]],
+        subject: str,
+        html: Optional[str] = None,
+        text: Optional[str] = None,
+    ):
+        from_email, from_name = sender
+        to_email, to_name = recipient
+        try:
+            message = Mail(
+                from_email=format_address(from_email, from_name),
+                to_emails=format_address(to_email, to_name),
+                subject=subject,
+                html_content=text)
```

Comment:
`html_content` argument should be assigned to the `html` variable. I [see there is a `plain_text_content`](https://github.com/sendgrid/sendgrid-python/blob/653cc47c54d53d401d65d1debb3cdc1d7905b4a1/sendgrid/helpers/mail/mail.py#L23-L32) that should be good for `text`.

So we should have:

```py
message = Mail(
    from_email=format_address(from_email, from_name),
    to_emails=format_address(to_email, to_name),
    subject=subject,
    html_content=html,
    plain_text_content=text,
)
```

Reply:
Ah, yes make sense.